packages/@angular/cli/commands/serve.ts

```diff
@@ -20,7 +20,8 @@ export interface ServeTaskOptions extends BuildOptions {
   host?: string;
   proxyConfig?: string;
   liveReload?: boolean;
-  liveReloadClient?: string;
```

Comment:
The option is setting the publicly accessible address of the host and is checked against the host header of all incoming requests.  It's also the name used within webpack-dev-server.